components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -261,7 +261,13 @@ data class NimbusAppInfo(
      *
      * Examples: "nightly", "beta", "release"
      */
-    val channel: String
+    val channel: String,
+    /**
+     * Some specific context that the SDK doesn't need to be explicitly aware of.
+     * this is used to allow targeting on fields that are plaform or app specific
+     * that other consumers of the SDK shouldn't consume
+     */
+    val appSpecificContext: Map<String, String>? = null
```

Comment:
We might also simplify a lot of code here by making all this non-optional.

```suggestion
    val appSpecificContext: Map<String, String> = mapOf()
```